src/views/Welcome/App.tsx

```diff
@@ -537,10 +590,11 @@ export default function App() {
             </div>
           </Card>
           <Spacer />
-          <Input.Password
+          <Input
             {...configPasswordInput.bindings}
+            type="password"
             placeholder="Enter your password to decrypt..."
-            width="100%"
+            style={{ width: "100%" }}
           />
         </Modal.Content>
         <Modal.Action onClick={() => loadConfigModal.setVisible(false)} passive>
```

Comment:
Please update the welcome page to use the Verto modals too @codingknite

Reply:
On it